src/components/course/data/hooks.js

```diff
@@ -305,6 +305,17 @@ export function useCourseEnrollmentUrl({
   );
 
   return enrollmentUrl;
-}
+};
+
+useCourseEnrollmentUrl.propTypes = {
+  catalogList: PropTypes.shape({}).isRequired,
+  enterpriseConfig: PropTypes.shape({}).isRequired,
+  key: PropTypes.string.isRequired,
+  location: PropTypes.shape({}).isRequired,
+  offers: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
```

Comment:
How does the 'isRequired' prop type interact with having a default argument? Is that fine?

Reply:
actually good catch, I should _not_ use isRequired here since I am supplying a default value